source/enable-autocomplete.txt

```diff
@@ -0,0 +1,98 @@
+.. _vs-code-enable-autocomplete:
+
+====================
+Enable Auto Complete
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can enable autocomplete for MongoDB Playgrounds. Enabling 
+autocomplete allows you to automatically populate database and collection 
+names when using ``use("")`` or 
+``db.getCollection("")`` syntax within a MongoDB playground.
+
+Before you Begin
+----------------
+
+Install the :ref:`MongoDB VSCode extension <vsce-install>`.
+
+Steps
+-----
+
+.. procedure::
+   :style: connected
+
+   .. step:: Open Settings.json
+     
+      In VSCode, open the command palette:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 30 70
+
+         * - Operating System
+           - Keyboard Command
+
+         * - Windows
+           - :kbd:`CTRL` + :kbd:`SHIFT` + :kbd:`P`
+
+         * - MacOS
+           - :kbd:`CMD` + :kbd:`SHIFT` + :kbd:`P`
+
+      Enter :guilabel:`Open User Settings (JSON)` and select the option 
+      from the autocompleted list:
+
+      .. figure:: /images/vsce-command-palette-json-settings.png
+         :figwidth: 700px
+         :alt: Image of a stream connection
+
+   .. step:: Add the autocomplete snippet to the ``Settings.json`` file
+     
+      Copy the following code snippet:
+
+      .. code-block::
+
+         "editor.quickSuggestions": {
+            "other": true,
+            "comments": false,
+            "strings": true
+         }
+
+      Paste the code into the ``Settings.json`` file:
+
+      .. figure:: /images/vsce-add-snippet-json-settings.png
```

Comment:
It's up to you, but I don't think this screenshot adds much value. I'd say we can delete it and just say "Paste the code into the ``Settings.json`` file.

Reply:
I do hate to include 1 additional screenshot myself, but the value add is the positioning of the snippet within the ``setting.json`` file. I am trying to reinforce with the screenshot that you can paste the snippet right before the closing ``}`` to alleviate any user confusion on where exactly it should go.